common/scripted_buttons/imperia_dead_man_buttons.txt

```diff
@@ -0,0 +1,153 @@
+﻿tur_republican_button = {
+	name = "republican_button"
+	desc = "republican_button_desc"
+
+	visible = {
+		owner = { 
+			has_variable = chose_initial_turk_government 
+		}
+	}
+
+	possible = {
+		trigger_if = {
+			limit = {
+				has_variable = has_selected_republican_once_var
+			}
+			custom_tooltip = {
+				text = cannot_choose_republican_twice_tt
+				always = no
+			}
+		}
+		any_interest_group = {
+			is_in_government = yes
+			OR = {	
+				leader = {
+					is_republican_supporter = yes
+				}
+				is_interest_group_type = ig_intelligentsia
+			}
+		}
+		custom_tooltip = {
+			text = not_already_supporting_republicanism
+			NOT = { has_variable = supporting_republican_var }
+		}
+	}
+	
+	selected = {
+		has_variable = supporting_republican_var
+	}
+
+	effect = {
+		set_variable = supporting_republican_var
+		set_variable = has_selected_republican_once_var
+		remove_variable = supporting_theocracy_var
+		remove_variable = supporting_communist_var
```

Comment:
These `remove_variable =` should be `remove_variable ?=` since it's not certain those variables are there. Trying to remove a variable that's not present results in an error.. But we're getting to the real minor stuff now :)